drydock/templates/drydock/k8s/celery/lms-worker.yml

```diff
@@ -13,8 +13,8 @@ spec:
             - --app=lms.celery
             - worker
             - --loglevel=info
-            - --concurrency=1
+            - --autoscale=10,1
```

Comment:
@Ian2012 as we agreed, we should remove the `autoscale` parameter

Reply:
I think is better to close the PR at the moment and revisit celery optimizations later.

As @Ian2012 said, removing the `autoscale` flag leaves the same command invocation as vanilla tutor. In that case is just easier to simply disable the plugin by default. Or delete the whole plugin (although maybe there's a situation on which the original plugin behavior is preferred?)